mmdet3d/evaluation/functional/panoptic_seg_eval.py

```diff
@@ -0,0 +1,386 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Dict, List, Tuple
+
+import numpy as np
+from mmengine.logging import MMLogger, print_log
+
+PQReturnsType = Tuple[np.double, np.double, np.ndarray, np.ndarray, np.ndarray]
+
+
+class EvalPanoptic:
+    r"""Evaluate panoptic results for Semantickitti and NuScenes.
+    Please refer to the `semantic kitti api
+    <https://github.com/PRBonn/semantic-kitti-api/>`_ for more details
+
+    Args:
+        classes (list): Classes used in the dataset.
```

Comment:
the order of the arguments in this class is inconsistent with those in the Metric, which indicates their importance and use frequencies. We might consider unify the orders of these arguments.
For example, if thing_classes/stuff_classes has higher priority in this class, then it should also have higher priority in PanopticSegMetric than min_num_points.